Bookish/Controllers/BookController.cs

```diff
@@ -0,0 +1,86 @@
+using System.Diagnostics;
+using Microsoft.AspNetCore.Mvc;
+using Bookish.Models;
+using Microsoft.AspNetCore.Authorization.Infrastructure;
+
+namespace Bookish.Controllers;
+
+public class BookController : Controller
+{
+    public IActionResult Catalogue()
+    {
+        var books = EditBooks.GetBookList();
+        return View(books);
+    }
+    
+    [HttpGet]
+    public IActionResult AddBook()
+    {
+        return View();
+    }
+    
+    [HttpPost]
+    public ActionResult AddBook(BookModel newBook)
+    {
+        EditBooks.InsertBooks(newBook);
+        return View(newBook);
+    }
+    [HttpGet]
+    public ActionResult EditBook(int id)
+    {
+        BookModel updateBook = EditBooks.GetBookFromId(id);
+        return View(updateBook);
+    }
+    
+    [HttpPost]
+    public ActionResult EditBook(BookModel replaceBook)
+    {
+        EditBooks.AlterBooks(replaceBook.Id, replaceBook);
+        return RedirectToAction(nameof(SearchBooksResults));
+    }
+    
+    [HttpPost]
+    public ActionResult DeleteBook(BookModel lostBook)
+    {
+        EditBooks.DeleteBooks(lostBook.Id);
+        return View();
+    }
+
+    [HttpGet]
+    public IActionResult SearchBooks()
+    {
+        return View();
+    }
+    
+    [HttpPost]
+    public IActionResult SearchBooks(BookModel search)
+    {
+        if (SearchBooksModel.IsFormBlank(search))
+        {
+            return View(search);
+        }
+
+        return RedirectToAction(nameof(SearchBooksResults), search);
+    }
+    
+    public IActionResult SearchBooksResults(BookModel search)
+    {
+        var results = SearchBooksModel.SearchForBook(search);
+        return View(results);
+    }
+
+    //[HttpPost]

```

Comment:
Why is this commented out? Should it have a different annotation?

Reply:
(This section of the code was heavily affected by what Max was working on in parallel - and is now outdated - so I mostly just left it alone in order to minimise merge conflicts later)

Reply:
What I mean is - this method should still have an annotation e.g. `[HttpGet]`